docs/docs/k8s/ingress.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Ingress Controller
+lang: en-US
+sidebarDepth: 1
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy oidc kubernetes Ingress reverse-proxy
+---
+
+# Kubernetes Ingress Controller
+
+Use Pomerium as a first-class secure-by-default Ingress Controller. Dynamically provision routes from Ingress resources and set policy based on annotations. By defining routes as Ingress resources you can independently create and remove them from Pomerium's configuration. This enables workflows more native to Kubernetes environments such as actions based on pull requests.
+
+## Prerequisites
+
+- A certificate management solution. If you do not already have one in place, this article covers using Cert Manager.
+- A Redis backend with high-persistence is highly recommended.
```

Comment:
Can we link to the redis docs?

Reply:
What does high-persistence mean?

Reply:
Linked to Redis and specifically their section on Persistence.